Source/Core/DolphinQt2/GameList/TableDelegate.cpp

```diff
@@ -28,18 +30,22 @@ void TableDelegate::paint(QPainter* painter, const QStyleOptionViewItem& option,
     DrawPixmap(painter, option.rect, Resources::GetCountry(data.toInt()));
     break;
   case GameListModel::COL_RATING:
-    DrawPixmap(painter, option.rect, Resources::GetRating(data.toInt()));
+    DrawPixmap(painter, option.rect, Resources::GetRating(data.toInt()), false);
     break;
   case GameListModel::COL_BANNER:
     DrawPixmap(painter, option.rect,
-               data.value<QPixmap>().scaled(NORMAL_BANNER_SIZE, Qt::KeepAspectRatio,
-                                            Qt::SmoothTransformation));
+               data.value<QPixmap>().scaled(NORMAL_BANNER_SIZE * scale, Qt::KeepAspectRatio,
```

Comment:
When `AA_EnableHighDpiScaling` is set, Qt operates in device-independent pixels, so just `NORMAL_BANNER_SIZE` is correct.

So, this should be scaled to `NORMAL_BANNER_SIZE`, not `NORMAL_BANNER_SIZE * scale` (or not scaled at all, I would think), and should be drawn to `NORMAL_BANNER_SIZE`, not `NORMAL_BANNER_SIZE * scale`.